frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -70,7 +70,26 @@ <h5 class="fw-light">Challenge Settings</h5>
         </mat-checkbox>
       </div>
     </div>
-
+    
+    <!-- Phase Edit Section-->
+    <div class="row settings-section">
+      <div class="col-lg-3 col-md-4 col-sm-3 col-xs-3">
+        <span class="fw-light fs-18">Edit Phase Detail</span>
```

Comment:
Can we please change `>Edit Phase Detail` to `Edit phase details:`? It would look consistent with `Edit challenge details:`

Reply:
Done!